alarm/models.py

```diff
@@ -66,4 +66,11 @@ def refresh_candle_data(cls, trade_pair, high_price, low_price):
         return candle
 
     def __str__(self):
-        return f"{self.trade_pair}: {self.low_price}-{self.high_price}"
+        return f"{self.low_price}-{self.high_price}"
+
+
+class ThresholdBrake(models.Model):
+    threshold = models.ForeignKey(Threshold, on_delete=models.CASCADE)
+
```

Comment:
With no happened at, how will you order threshold by happened date?

Reply:
I deleted it because we haven't used it up to this point. I returned that field back. And what interval will we filter threshold information with the happend_at field?